pkg/oauthserver/server/session/authenticator.go

```diff
@@ -35,40 +34,36 @@ func NewAuthenticator(store Store, maxAge int32) *Authenticator {
 func (a *Authenticator) AuthenticateRequest(req *http.Request) (user.Info, bool, error) {
 	values := a.store.Get(req)
 
-	expires, ok, err := values.GetInt64(expKey)
-	// TODO in a release when mixed masters are no longer an issue, replace with:
-	// if !ok || err != nil {
-	if err != nil {
-		return nil, false, err
-	}
+	expires, ok := values.GetInt64(expKey)
 
 	// TODO drop this logic in a release when mixed masters are no longer an issue
 	if !ok {
-		expiresString, ok, err := values.GetString(expiresKey)
-		if !ok || err != nil {
-			return nil, false, err
+		// TODO replace with:
+		// return nil, false, nil
+
+		expiresString, ok := values.GetString(expiresKey)
+		if !ok {
+			return nil, false, nil
 		}
-		expires, err = strconv.ParseInt(expiresString, 10, 64)
+		expiresParse, err := strconv.ParseInt(expiresString, 10, 64)
 		if err != nil {
-			return nil, false, fmt.Errorf("error parsing expires timestamp: %v", err)
+			return nil, false, nil
```

Comment:
if we're not going to return an error here, we can remove `error` from the function return signature all together. (should at least debug-level log the error though)

Reply:
You never know; things go wrong in mysterious ways  (especially when other people start messing with the code 🙂).

Ack on the interface